wrapper.py

```diff
@@ -310,8 +310,9 @@ def _load_custom_shape(f):
     """
     data_reshaped = []
     counter = 0
```

Comment:
The solution you suggested, doesn't work here, because `counter` has to take the accumulated values of `f.attrs['oldshape']`.
For Python >= 3.2, there is `itertools.accumulate` that delivers this behavior. For Python 2, no ready-made solution exists, so I added a helper function.
Once we implement support for Python 3, we can built in a check for this.